addons/cswCompat/script_component.hpp

```diff
@@ -1,5 +1,5 @@
 #define COMPONENT cswCompatCUP
-#include "\z\ace\addons\main\script_mod.hpp"
+#include "\z\potato\addons\core\script_mod.hpp"
```

Comment:
Some old loadouts that might still be in use are expecting `ace_cswCompatCUP_cup_m2`
ref https://github.com/BourbonWarfare/bwmf/pull/178/files